### ISSUE TITLE
Simplify the Aggregation type

### ISSUE THREAD

**Issue by shaunsales** [RedisTimeSeries/NRedisTimeSeries#32]
Working with the Aggregation type is currently not as easy as it could be. We can avoid creating a new object, and the ambiguity with the Aggregation.Name property (easy to confuse when comparing). 

I propose we convert it to an enum, which will be easier for the end user;
```
public enum Aggregation
{
	Avg,
	Sum,
	Min,
	Max,
	Range,
	Count,
	First,
	Last,
	StdP,
	StdS,
	VarP,
	VarS,
}
```
And internally use an extension method to retrieve the RedisTimeSeries param;
```
internal static class AggregationExtensions
{
	public static string Param(this Aggregation aggregation) => aggregation switch
	{
		Aggregation.Avg => "avg",
		Aggregation.Sum => "sum",
		Aggregation.Min => "min",
		Aggregation.Max => "max",
		Aggregation.Range => "range",
		Aggregation.Count => "count",
		Aggregation.First => "first",
		Aggregation.Last => "last",
		Aggregation.StdP => "std.p",
		Aggregation.StdS => "std.s",
		Aggregation.VarP => "var.p",
		Aggregation.VarS => "var.s",
		_ => throw new ArgumentException("Invalid Aggregation parameter"),
	};
}
```

**Comment by DvirDukhan** [comment]
@shaunsales thanks, good point

@ashtul @filipecosta90 good first issue

**Comment by DvirDukhan** [comment]
closed by #36
